### PR TITLE
Bump quote to 0.5

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

File: tests/expectations/tests/issue-1040.rs [hunk 1]
```diff
@@ -1,7 +1,5 @@
 /* automatically generated by rust-bindgen */
 
-
 #![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
 
-
-pub const g_107: ::std::os::raw::c_ulonglong = 18446744073709551615;
+pub const g_107: ::std::os::raw::c_ulonglong = -1;
```
Comment (Eijebong) [inline tests/expectations/tests/issue-1040.rs]: This is because proc-macro2 casts ints as i64 if they aren't suffixed

Comment (dtolnay) [reply tests/expectations/tests/issue-1040.rs]: This will be fixed after https://github.com/rust-lang/rust/pull/49597 has landed.

Comment (Eijebong) [reply tests/expectations/tests/issue-1040.rs]: Does that mean another proc-macro2 breaking change ?

Comment (Eijebong) [reply tests/expectations/tests/issue-1040.rs]: Also this doesn't work for u32 soooo....

File: tests/expectations/tests/comment-indent.rs [hunk 2]
```diff
@@ -90,9 +77,7 @@ pub mod root {
                 )
             );
         }
-        /// I'm in an inline namespace, and as such I shouldn't get generated inside
-        /// a rust module, except when the relevant option is specified. Also, this
-        /// comment shouldn't be misaligned.
+        #[doc = " I\'m in an inline namespace, and as such I shouldn\'t get generated inside\n         a rust module, except when the relevant option is specified. Also, this\n         comment shouldn\'t be misaligned."]
```
Comment (emilio) [inline tests/expectations/tests/comment-indent.rs]: This is not great :(

File: tests/expectations/tests/comment-indent.rs [hunk 3]
```diff
@@ -111,7 +96,7 @@ pub mod root {
                 concat!("Alignment of ", stringify!(InInlineNS))
             );
         }
-
+        #[doc = ""]
```
Comment (emilio) [inline tests/expectations/tests/comment-indent.rs]: This should check whether the comment is empty.

File: src/ir/comment.rs [hunk 4]
```diff
@@ -54,7 +54,7 @@ fn preprocess_single_lines(comment: &str, indent: usize) -> String {
             let indent = if is_first { "" } else { &*indent };
             is_first = false;
             let maybe_space = if l.is_empty() { "" } else { " " };
-            format!("{}///{}{}", indent, maybe_space, l)
+            format!("{}{}{}", indent, maybe_space, l)
```
Comment (emilio) [inline src/ir/comment.rs]: If we end up going for `#[doc]` stuff, all the maybe_space stuff can go away, but I don't see how multiline stuff works now, it seems to mangle the documentation, which I think it's not acceptable.

File: tests/expectations/tests/comment-indent.rs (hunk 2 above)
Comment (Eijebong) [reply tests/expectations/tests/comment-indent.rs]: I know :(

**Pull request merged**